doc/developingCubeBrowsers/interactions_2.ipynb

```diff
@@ -0,0 +1,244 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Linking up Cube Pickers and Plot Customization Options\n",
+    "\n",
+    "This is an example of a set of workflow options linked up with ipywidgets.  The options in this demo are for preparing one plot of an iris cube.\n",
+    "\n",
+    "There are some assumptions in this workflow:\n",
+    "1. Data path is set to iris.sample_data_path\n",
+    "2. Coordinates for plot axes will be longitude vs latitude\n",
+    "3. Plot type will be line contour (plot customizations are applicable to this plot type)\n",
+    "\n",
+    "This notebook demostrates the links between some elements of a selection process."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "import IPython.display\n",
+    "import ipywidgets\n",
+    "import iris\n",
+    "import cube_browser\n",
+    "import glob\n",
+    "import os"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# This filepath is a directory containing the set of options for the first selection box.\n",
+    "file_options = glob.glob(os.path.join(iris.config.SAMPLE_DATA_DIR, '*')) \n",
+    "\n",
```

Comment:
@marqh This (above) is what I used instead of iris.sample_data_path() on Bill's advice.